classes/PodsRESTHandlers.php

```diff
@@ -59,58 +59,64 @@ public static function get_handler( $object, $field_name, $request ) {
 		$id       = pods_v( 'id', $object );
 		$pod      = self::get_pod( $pod_name, $id );
 		if ( $pod && PodsRESTFields::field_allowed_to_extend( $field_name, $pod, 'read' ) ) {
+
 			$params     = null;
 			$field_data = $pod->fields( $field_name );
 			if ( 'pick' == pods_v( 'type', $field_data ) ) {
+
 				$output_type = pods_v( 'rest_pick_response', $field_data['options'], 'array' );
 				if ( 'array' == $output_type ) {
 
 					$related_pod = $pod->field( $field_name, array( 'output' => 'pod' ) );
-					$fields      = $related_pod->fields();
-					$fields      = array_keys( $fields );
-					if ( isset( $related_pod->pod_data['object_fields'] ) && ! empty( $related_pod->pod_data['object_fields'] ) ) {
-						$fields = array_merge( $fields, array_keys( $related_pod->pod_data['object_fields'] ) );
-					}
+					if ( is_a( $related_pod, 'Pods' ) ) {
```

Comment:
Either that or false, if there are no related items afaik.  So this "fix" is just totally bypassing the code inside the `if`, since I don't know of a situation where a pick field is going to return a singular Pod object here.  Needs a re-think, including checking if the related object is actually exposed to the REST API.